css/style.css

```diff
@@ -0,0 +1,234 @@
+body {
+  background-color: #eee;
+  display: flex;
+  flex-direction: column;
+  font-family: sans-serif;
+  color: #000;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  background-color: #eee;
+  display: grid;
+  grid-template-columns: 300px 1fr;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  padding-top: 10px;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  max-width: 750px;
+  position: relative;
+  top: 40px;
+  z-index: 0;
+}
+
+#header-img {
+  grid-column: 1;
+  width: 100%;
+  align-self: center;
+}
+
+#nav-bar {
+  grid-column: 2;
+  align-self: center;
+  justify-self: right;
+  display: flex;
+  margin-right: 5px;
+}
+
+.nav-link {
+  text-decoration: inherit;
+  color: inherit;
+  display: inline-block;
+  margin: 5px;
+}
+
+/* input0[type=submit] {
+      text-align: center;
+  } */
+
+#intro {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#intro h2 {
+  margin-bottom: 15px;
+}
+
+#form {
+  display: grid;
+  align-items: center;
+  grid-template-rows: 1fr 1fr;
+  grid-gap: 10px;
+  margin-bottom: 30px;
+}
+
+input[type="submit"] {
+  width: 200px;
+  background-color: #4caf50;
+  padding: 0 32px;
+  border-style: none;
+  border-radius: 3px;
+  color: #fff;
+}
+#features {
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.feature-section {
+  display: grid;
+  grid-template-columns: 100px 1fr;
+  align-items: center;
+}
+
+.feature-img {
+  width: 50%;
+  justify-self: center;
+  align-self: start;
+  margin-top: 20px;
+}
+
+#video {
+  margin: auto;
+  max-width: 100%;
+  margin-top: 20px;
+  margin-bottom: 10px;
+}
+
+#products {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 10px;
+  margin-top: 20px;
+}
+
+.product-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-style: groove;
+  border-color: lightgray;
+  border-width: 1px;
+  padding-bottom: 10px;
+}
+
+.product-header {
+  margin-bottom: -20px;
+  background-color: #eee;
+}
+
+.product-price {
+  font-size: 130%;
+  font-weight: bold;
+  text-align: center;
+}
+
+.product-desc {
+  list-style: none;
+}
+
+footer {
+  background-color: #eee;
+  margin-top: 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  padding: 10px;
+  padding-bottom: 0px;
+}
+
+#footer-nav {
+  display: flex;
+  grid-gap: 10px;
+}
+
+footer p {
+  font-size: 85%;
+  color: gray;
+  margin-bottom: 10px;
+}
+
+.btn {
+  text-transform: uppercase;
+  background-color: hsl(45, 50%, 80%);
+  font-weight: bold;
+  width: 150px;
+  /* border-radius: 6px; */
```

Comment:
Kindly, **remove unused code as a comment in your codebase** as It will make your codebase cleaner and easy to follow

Reply:
Good job 👍 in identifying that comments should be removed.